### PR TITLE
fix(files): view-in-folder and bulk-tagging enabled conditions

### LINKED ISSUE #47115: [Bug]: Deleted files offer not-working "view-in-folder" action
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

The file actions for deleted files offer the file action "view in folder" but that action does not work, it will redirect to files where of course no file is visible.

Is it intended to be called "Show source folder" or similar? Or should it not be shown at all?

image

### Steps to reproduce

1. upload a file
2. delete the file
3. go to trash bin
4. click file actions

### Expected behavior

Either name it "Show source folder" or do not show it at all.

### Installation method

None

### Nextcloud Server version

master

### Operating system

None

### PHP engine version

None

### Web server

None

### Database engine version

None

### Is this bug present after an update or on a fresh install?

None

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

_No response_

### List of activated Apps

_No response_

### Nextcloud Signing status

_No response_

### Nextcloud Logs

_No response_

### Additional info

cc @skjnldsv

### PR DESCRIPTION
**Pull request by skjnldsv**
Fix https://github.com/nextcloud/server/issues/47115

### CODE REVIEW

**Comment by skjnldsv** [comment]
Seems this ain't the only issues
image

File: apps/systemtags/src/files_actions/bulkSystemTagsAction.ts [hunk 1]
```diff
@@ -38,8 +38,18 @@ export const action = new FileAction({
...
 
-		// If the user is not logged in, the action is not available
-		return true
+		// Disabled for non dav resources
+		if (nodes.some((node) => !node.isDavRessource)) {
+			return false
+		}
+
+		// Disabled if node outside user root folder
+		if (nodes.some((node) => !node.root?.startsWith('/files'))) {
```
Comment (susnux) [inline apps/systemtags/src/files_actions/bulkSystemTagsAction.ts:47]: Why this? For view in folder it makes sense, but tagging should work for all nodes

**Review by susnux — APPROVED** [review]

**Review by Pytal — APPROVED** [review]

**Pull request merged**